Sources/Prometheus/MetricTypes/Metric.swift

```diff
@@ -1,34 +1,40 @@
 /// Different types of metrics supported by SwiftPrometheus
 public enum MetricType: String {
```

Comment:
Should also follow the Prom prefixing, so `PromMetricType`

Reply:
why don't we do `PrometheusMetricType`?

Reply:
`Prom` is kind of "well known short name" for prometheus weirdly enough but yeah we could do full names everywhere if we'd want to hm... consistency is king tho :)